src/services/utilities/utilities.svc.coffee

```diff
@@ -19,15 +20,38 @@ angular
       maxDate = _.max _.map(dates, (d) -> moment(d))
       return [minDate.startOf('day').toDate(), maxDate.startOf('day').toDate()]
 
-    getPeriodWord = (period) ->
+    ###
```

Comment:
@cesar-tonnoir not auto, but Im going to make it a convention to ensure doc on public methods following this conventions: http://usejsdoc.org/